src/openms_gui/include/OpenMS/VISUAL/SpectrumCanvas.h

```diff
@@ -62,6 +62,110 @@ namespace OpenMS
 {
   class SpectrumWidget;
 
+  class LayerStack
```

Comment:
move implementation to cpp file.
maybe add a line of comment that layers are automatically sorted by layer type?